test/functional/wiki_controller_test.rb

```diff
@@ -132,11 +132,26 @@ def teardown
     assert_equal flash[:notice], "Edits saved."
   end
 
+  test "basic user blocked from updating a locked wiki page" do
+    node(:organizers).add_tag('locked', rusers(:admin)) # lock the page with a tag
+    # then try updating it
+    assert_difference 'DrupalNodeRevision.count', 0 do
+      post :update,
```

Comment:
Oh, note that your changes were to the `:edit` action, but your test is for the `:update` action. Both need to be changed and tested, but I think you're changing one but testing the other in this step.